### DISCUSSION TITLE
Public servers?

### DISCUSSION THREAD

**Discussion by Void-025** [StoneLabs/strains-of-ascension#13] [Q&A]
I could just setup a local server to play it myself, but seeing as it's server-only in the first place it seems like it might be intended to be used primarily in multiplayer? If so, are there any public/"official" servers to play it on?

**Comment by StoneLabs** [comment]
I don't think there are any. At least I don't know of any.

I mainly developed it as a server only to host it with my friends so I recommend you set up a local server, sry ^^

(Ah be advised: after some time I realized the nether is insanely hard on default settings so feel free to disable strains there with commands if you'd like)
